src/styles/calendar.scss

```diff
@@ -0,0 +1,23 @@
+.calendar {
```

Comment:
place your calendar at the center of the page
<img width="842" alt="Screenshot 2023-12-19 at 21 10 31" src="https://github.com/mate-academy/layout_calendar/assets/117587463/2b562fc1-df9a-4b63-8ac0-6b0238b7b251">